JSDemos/Demos/PivotGrid/SummaryDisplayModes/description.md

```diff
@@ -1 +1,19 @@
-The PivotGrid component supports several modes for summary calculation, such as&nbsp;a&nbsp;variation or&nbsp;percent of&nbsp;a&nbsp;total value. Select the required mode from the context menu appearing when you right-click the &laquo;Relative Sales&raquo; field.
\ No newline at end of file
+Summaries are values displayed in pivot grid cells. You can configure summaries in the [fields](/Documentation/ApiReference/Data_Layer/PivotGridDataSource/Configuration/fields/) that belong to the *"data"* area (`Relative Sales` in this demo). 
+
+## Available Summary Display Modes
+You can specify a post-processing function to modify summaries after they are calculated. Use the [summaryDisplayMode](/Documentation/ApiReference/Data_Layer/PivotGridDataSource/Configuration/fields/#summaryDisplayMode) property to specify how to display summaries. This property allows you select one of the predefined summary display modes. To implement a custom mode, use the [calculateSummaryValue](/Documentation/ApiReference/Data_Layer/PivotGridDataSource/Configuration/fields/#calculateSummaryValue) property. Note that you cannot use predefined and custom modes simultaneously. 
+
+The list below shows available predefined modes. These modes calculate summaries depending on previous row values or totals:
+
+- *"absoluteVariation"*
+- *"percentVariation"*
+- *"percentOfColumnTotal"*
+- *"percentOfRowTotal"*
+- *"percentOfColumnGrandTotal"*
+- *"percentOfRowGrandTotal"*
+- *"percentOfGrandTotal"*
```

Comment:
A user can click the field in the "data" area and see these modes. They are also available in the docs.
```suggestion
```